shared/reducers/__tests__/fs.js

```diff
@@ -184,12 +184,19 @@ describe('fs reducer', () => {
 
   test('favorritesLoaded: reuse tlf', () => {
```

Comment:
whoa I didn't know about these tests

Reply:
Yep! For Fs we only have them for the reducer though as they tend to be complex and error prone.